spec/controllers/services_controller_spec.rb

```diff
@@ -1,83 +1,94 @@
-# -*- encoding : utf-8 -*-
```

Comment:
why has this been removed?

Reply:
These aren't needed in Ruby 2.0+ and as #4209 is closed I think its fine to get rid of these. One of RuboCop checks I was going to suggest we enable would enforce getting rid of these lines.